docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following issues.
+
+.Lack of Unique IDs and Aria-Labels on Sections/Categories, Hindering Automation
```

Comment:
Style guide - use of capitals in titles

Reply:
See IBM Style guide, "Headings, titles, and banners" (page reference varies with edition, so I'm citing the section).